tools/deploy_ipk.sh

```diff
@@ -28,4 +36,33 @@ rm -rf /opt/prplmesh
 opkg install -V2 "$DEST_FOLDER/$IPK_FILENAME"
 EOF
 
-echo "Done"
+    if [ "$CERTIFICATION_MODE" = true ] ; then
+        echo "Certification mode will be enabled on the target"
+        ssh "$TARGET" "uci set prplmesh.config.certification_mode=1 && uci commit"
+        echo "Certification mode enabled on the target."
+    fi
+    echo "Done"
+}
+
+main() {
+        OPTS=$(getopt -o 'h' --long help,certification-mode -n 'parse-options' -- "$@")
+
+    if [ "$?" != 0 ] ; then err "Failed parsing options." >&2 ; usage; exit 1 ; fi
+
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            --certification-mode)
```

Comment:
why not add also short option?

Reply:
I can convert it to a short option if you want. 
I find having both a short option and a long option for everything is more work (and in the end we mess up the options by forgetting to update one or the other), and doesn't bring much advantages (we end up always using only one of the 2).